src/leetCodeExecutor.ts

```diff
@@ -87,6 +89,11 @@ class LeetCodeExecutor {
         return filePath;
     }
 
+    public async showSolution(node: IProblem, language: string): Promise<string> {
```

Comment:
rename the param `node` to `problemNode` or `problem` to avoid get confusing with the command `node`